pkg/trace/config/config.go

```diff
@@ -588,3 +598,9 @@ func (c *AgentConfig) AllFeatures() []string {
 	}
 	return feats
 }
+
+func inAzureAppServices(getenv func(string) string) bool {
+	exists_linux := getenv(RunZip)
+	exists_win := getenv(AppLogsTrace)
+	return len(exists_linux)+len(exists_win) > 0
```

Comment:
Realized this isn't the most readable, should I add a helper like `hasEnv()` or is this okay?